testng-eclipse-plugin/src/main/org/testng/eclipse/buildpath/BuildPathSupport.java

```diff
@@ -27,7 +27,7 @@
 
 public class BuildPathSupport {
 
-  private static final PluginDescription TESTNG_PLUGIN_DESC = new PluginDescription("org.testng", new VersionRange("[6.0, 7.0]"), null, null, "org.testng.source", null);
+  private static final PluginDescription TESTNG_PLUGIN_DESC = new PluginDescription("org.testng", new VersionRange("[6.0, 8.0)"), null, null, "org.testng.source", null);

```

Comment:
somehow, this range `(6.0, 7.0]` doesn't work for 7.0.0 release, make the range wider.